Makefile

```diff
@@ -92,7 +93,7 @@ manifests: controller-gen generate-daemon-manifest ## Generate WebhookConfigurat
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
 .PHONY: generate
-generate: controller-gen ebpf-generate ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

Comment:
why removed the ebpf-generate ?

Reply:
Because it's always rebuilding the .o files and with every commit I have to unstage the objects. It's also not necessary, I thought we'd prebuild once, check in, and then rebuild when needed. I'm not sure if the .gitignore for that works from my first commit by the way, seems if a file is in once, git will still consider it:

Reply:
quick follow-up: it's make sense to run this as a pre-requisite of the daemon-set build I imagine; but do we need the .o files inside the repo, then?